src/felt.rs

```diff
@@ -69,61 +69,61 @@ impl<const MODULUS: u64> FeltTrait for Felt<MODULUS> {
 mod tests {
     use super::*;
 
-    type Felt17 = Felt<17>;
+    type FeltX = Felt<17>;
```

Comment:
This literally a `Felt17` so I don't think we should change the name for the tests